backend/src/controllers/items-controller.ts

```diff
@@ -0,0 +1,136 @@
+import { Request, Response } from 'express';
+import { validationResult } from 'express-validator';
+import Item from '../models/item';
+import User from '../models/user'
+import InventoryItem from "../models/inventory-item";
+import mongoose from 'mongoose';
+import {ObjectId} from "mongodb";
+
+//Creates an item
+export const createItem = async (req: Request, res: Response) => {
+    //Validation check
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+        console.log("errors", errors);
+        return res.status(400).json({errors: errors.array()});
+    }
+    //Duplicated item check
+    const dupeItem = Item.findOne({name: req.body.name})
+    if (!dupeItem) {
+        return res.status(409).json({error: "An item with the same name already exists."});
+    }
+    //Checks if the ID matches the owner of the item
+    const itemOwnerId = req.body._id;
```

Comment:
```suggestion
    const itemOwnerId = req.body.itemOwnerId;
```
So that those calling the API receive the`itemOwnerId`

Reply:
I get this error when using `_id` in the body:

![image](https://github.com/miguelaenlle/merch-order-request-form/assets/66172204/fd5cf032-f5be-4f3b-ae65-09bc9a52427f)

The above suggestion will fix that.